bin/meli-test-account.js

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env node
+require('dotenv').config({path: require('path').resolve(process.cwd(), '.env.test')})
+require('../config/db').connect()
+
+const app = require('express')()
+const req = require('request-promise')
+const chromeLauncher = require('chrome-launcher')
+const {Spinner} = require('cli-spinner')
+
+const auth = require('../routes/auth')
+const Account = require('../model/account')
+const {refresh} = require('../config/meli-auth')
+
+const devAccountUsername = process.env.DEV_ACCOUNT_USERNAME
+const port = process.env.PORT
+const TIMEOUT_MS = 60 * 1000
+
+const global = {
+  spinner: null
+}
+
+async function getDevAccount() {
+  const account = await Account.findOne({nickname: devAccountUsername})
+  if (!account) {
+    throw new Error(`Dev Account username ${devAccountUsername} not found.`)
+  }
+  if (!account.isAuthorized()) {
+    const accessToken = await refresh.requestNewAccessToken('mercadolibre', account.auth.refreshToken)
+    await account.updateAccessToken(accessToken)
+  }
+  return account
+}
+
+async function requestTestAccount(devAccount) {
+  const testAccountRequestOptions = {
+    method: 'POST',
+    uri: 'https://api.mercadolibre.com/users/test_user',
+    qs: {
+      access_token: devAccount.auth.accessToken
+    },
+    body: {
+      site_id: 'MLA'
+    },
+    json: true // Automatically stringifies the body to JSON
+  }
+  return req(testAccountRequestOptions)
+}
+
+function onAuthSuccess() {
+  global.spinner.stop(true)
+  console.log('Test account Register success! Exiting...')
+  process.exit()
+}
+
+function onAuthAbort() {
+  global.spinner.stop(true)
+  console.log('Chrome login window closed.')
+  process.exit()
+}
+
+async function launchChrome(loginUrl) {
+  console.log(`Waiting for login on: ${loginUrl}`)
+  const chrome = await chromeLauncher.launch({
```

Comment:
You should use [puppeteer](https://github.com/GoogleChrome/puppeteer) so you can use an nested chromium on your nodejs app instead of depending of the OS chrome, also this simplifies the usage of the CLI: users don't need to have chrome installed. Run it on `headless:false` mode so you can wait the user to enter the login credentials

Reply:
Yes, I noticed the puppeteer alternative, but I thought it would be an overkill "just" for opening a chrome tab to input MercadoLibre credentials and then close it. It's not even a requirement for the CLI to work: it there is no Chrome installed (or maybe if a browser is not a user preference at all), the login can be made by manually navigating to `loginUrl` in any other browser. 
However there is no error handling being made if there is no Chrome installation and probably the app would break,.. I should work on that at least